### PR TITLE
Release 0.20.2 (956)

### LINKED ISSUE #4572: unable to play any video: "YouTube did not provide player config even after three attempts" | "Could not parse website" error on every video [APK LINKED IN COMMENT]
# TEMP Debug APK: https://github.com/TeamNewPipe/NewPipe/pull/4675#debug-apk-hotfix
We'll release a new version
#  

Newpipe stopped working, probably some changes server-side on YouTube? I don't known, suddenly I can't play any video (not even touched my phone today and yesterday was playing fine. Enough talking here, the error report:

## Exception
* __User Action:__ requested stream
* __Request:__ https://www.youtube.com/watch?v=wPsom6R8rvo
* __Content Country:__ US
* __Content Language:__ en-US
* __App Language:__ en_US
* __Service:__ YouTube
* __Version:__ 0.20.1
* __OS:__ Linux Android 7.1.1 - 25
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.exceptions.ParsingException: YouTube did not provide player config even after three attempts
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.onFetchPage(YoutubeStreamExtractor.java:664)
	at org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:56)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:68)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:64)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:116)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$5fJcha6Sq5APJBLdG6osaJby-mc.call(lambda)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14935)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
	at java.lang.Thread.run(Thread.java:761)

```
</details>
<hr>

I don't think this is a crash

### PR DESCRIPTION
**Pull request by B0pol**
#### What is it?
- [x] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

Hotfix v0.20.2

TODO: update extractor version when TeamNewPipe/NewPipe#441 is merged

#### Fixes the following issue(s)

- <span id="debug-apk-hotfix">fixes</span> #4572 

#### Relies on the following changes

- TeamNewPipe/NewPipeExtractor#441
#### APK testing 

<a href="https://github.com/TeamNewPipe/NewPipe/files/5445521/newpipe-HOTFIX.zip">newpipe-HOTFIX.zip</a>

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: fastlane/metadata/android/en-US/changelogs/956.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+[YouTube] Fixed crash when loading video
```
Comment (opusforlife2) [inline fastlane/metadata/android/en-US/changelogs/956.txt:1]:
```suggestion
[YouTube] Fixed crash when loading any video
```

**Comment by opusforlife2** [comment]
Yup. That's how it's usually done. And then master is merged into dev.

**Comment by Stypox** [comment]
> A hotfix allows work on the master branch instead of the dev branch.
https://teamnewpipe.github.io/documentation/05_releasing/

**Comment by B0pol** [comment]
yes it should, my bad

**Comment by B0pol** [comment]
done. It was already based on master, but by default it selected dev on github

**Review by TobiGr — APPROVED** [review]

**Comment by falaafel** [comment]
Can someone give me steps to install the new version and to get it running nicely?

**Comment by B0pol** [comment]
https://newpipe.schabi.org/blog/announcement/f-droid/pinned/f-droid-repo/

**Comment by vkay94** [comment]
Does the current `dev` branch work for you? I've tried installing the original `dev` (after errors in rebased builds) but I get the "three attempts"-error again. I've noticed that the `master` branch has a different extractor version which works without problems.

**Comment by B0pol** [comment]
No, it does not. `master` has not been merged to dev, then you should do it locally before. There will be merge conflicts, change newpipeextractor version to commit hash [here](https://github.com/TeamNewPipe/NewPipeExtractor/commits/dev)

@TobiGr @TheAssassin (maybe?) please merge master into dev and update extractor version

**Comment by vkay94** [comment]
@B0pol  Yeah, I did that locally, I just though that the TeamNewPipe version in `dev` contains the fix since the hash now differs from before 😄

**Comment by Andre601** [comment]
> https://newpipe.schabi.org/blog/announcement/f-droid/pinned/f-droid-repo/

Doesn't really explain the actual updating process...
It says to deinstall newpipe to then update the source packages on F-Droid???

**Pull request merged**